### PR TITLE
feat: replace K0S/K0L with K0/K0bar

### PR DESCRIPTION
**Pull request by redeboer**
Use the mass eigenstates of K0 instead of K0S and K0L

### CODE REVIEW

**Comment by redeboer** [comment]
@spflueger 
Have a look: d276c4d36a99fbf09ed964afa3743bf53b88fba9

File: expertsystem/particle_list.yml [hunk 1]
```diff
@@ -365,23 +365,23 @@ ParticleList:
...
-    Mass: 0.497614
+  K0:
+    PID: 311
+    Mass: 0.497611
     QuantumNumbers:
       Spin: 0
       Charge: 0
       Parity: -1
-      IsoSpin: { Value: 0.5, Projection: 0.0 }
+      IsoSpin: { Value: 0.5, Projection: -0.5 }
```
Comment (spflueger) [inline expertsystem/particle_list.yml:375]: Missing `Strangeness: 1`

File: expertsystem/particle_list.yml [hunk 2]
```diff
@@ -365,23 +365,23 @@ ParticleList:
...
-    Mass: 0.497614
+  K0bar:
+    PID: -311
+    Mass: 0.497611
     QuantumNumbers:
       Spin: 0
       Charge: 0
       Parity: -1
-      IsoSpin: { Value: 0.5, Projection: -0.5 }
+      IsoSpin: { Value: 0.5, Projection: 0.5 }
```
Comment (spflueger) [inline expertsystem/particle_list.yml:384]: Missing `Strangeness: -1`

File: tests/channels/test_d0_to_kskpkm.py [hunk 3]
```diff
@@ -12,7 +12,7 @@ def test_script():
     logging.getLogger().setLevel(logging.INFO)
     # initialize the graph edges (initial and final state)
     initial_state = [("D0", [0])]
-    final_state = [("K_S0", [0]), ("K+", [0]), ("K-", [0])]
+    final_state = [("K0bar", [0]), ("K+", [0]), ("K-", [0])]
```
Comment (spflueger) [inline tests/channels/test_d0_to_kskpkm.py:15]: why you took `K0bar` and not `K0`?

Comment (redeboer) [reply tests/channels/test_d0_to_kskpkm.py:15]: Eamm, random choice

Comment (spflueger) [reply tests/channels/test_d0_to_kskpkm.py:15]: I see, `K0` would have been more straight forward for me. But ok I wont be that picky

**Pull request merged**